detection-rules/impersonation_meta.yml

```diff
@@ -126,12 +131,17 @@ source: |
       profile.by_sender().prevalence != "common"
       and not profile.by_sender().solicited
     )
+    or sender.email.domain.domain in $replyto_service_domains
```

Comment:
Did you want to combine this with the `beta.profile.by_reply_to() logic?

Reply:
I ripped it out because of the Appsheet behaviour.